packages/relay-runtime/store/__tests__/resolvers/HelloWorldResolverWithContext.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict-local
+ * @format
+ * @oncall relay
+ */
+
+'use strict';
+
+import type {LiveResolverContextType} from '../../../mutations/__tests__/LiveResolverContextType';
```

Comment:
Nit: For clarity sake, I think it would be good to rename this type to self-document that it's a test fixture representing the host's context typing, not a generic Relay provided context typing.